### ISSUE TITLE
Posterior tibial tendon injury

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11176]
**Preferred term label:**
Posterior tibial tendon injury

**Synonyms**
https://www.ncbi.nlm.nih.gov/medgen/C1136154 : Dysfunction of posterior tibial tendon

**Definition (free text, please give PubMed ID)**
MRI or ultrasound of the ankle/foot showing posterior tibial tendon degeneration such as tendinosis, tenosynovitis, edema around the tendon, or tendon tear

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posterior tibial tendon insufficiency (PTTI)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001398
- label: Posterior tibial tendon injury
- parents: [Abnormal foot morphology (HP:0001760)](https://hpo.jax.org/app/browse/term/HP:0001760)
- pmid: PMID:28979585|PMID:31194317
- def: Damage to the posterior tibialis tendon, which runs along the inside of the ankle and foot, supporting the arch of the foot. MRI or ultrasound of the ankle/foot shows posterior tibial tendon degeneration such as tendinosis, tenosynovitis, edema around the tendon, or tendon tear.
- comment: The posterior tibialis muscle originates from the posterior proximal tibia, interosseous membrane and proximal fibula; it is in the deep posterior compartment of the leg and turns at the medial malleolus, changing its line of pull from the vertical one to the horizontal one. The myotendinous junction is situated at the distal third of the leg and only the tendinous portion turns at the medial malleolus; anterior subluxation is prevented by a flexor retinaculum which binds the tendon into the shallow groove at the medial malleolus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11176
- synonyms:
